lingvo/core/beam_search_helper.py

```diff
@@ -87,11 +87,12 @@ def InitBeamSearchState(theta, encoder_outputs, num_hyps_per_beam):
           num_hyps_per_beam: An int, number hyps to keep for source sentence.
         Returns:
           initial_results: a `.NestedMap` of initial results. It should contain
-              the following tensors at the minimum.
-                  log_probs: The initial log probs for each of the tokens in
-                      the target vocab, of shape [tgt_batch, vocab_size].
-                  atten_probs: The initial attention probs, of shape [tgt_batch,
-                      src_len].
+              the following tensors at minimum:
+              .log_probs: The initial log probs for each of the tokens in
+                  the target vocab, of shape [tgt_batch * num_hyps_per_beam,
```

Comment:
The change to "tgt_batch * num_hyps_per_beam" is not correct, in two ways: (1) tgt_batch is already num_hyps_per_beam * src_batch; (2) it's actually (num_hyps_per_beam * src_batch) rather than (src_batch * num_hyps_per_beam), that is, src_batch "b" and hyp_per_beam "h" is represented at index (h * src_batch + b).

More generally, the request combines formatting changes with content changes. It's best to separate the changes so that the content changes are more visible.

Reply:
I change the commit only format the doc, but there is not definition of "tgt_batch" .

Reply:
I still see "tgt_batch * num_hyps_per_beam" on L92 and L95.

Reply:
I have fixed it, are you sure?